app/src/main/java/com/github/swent/echo/data/SampleEvents.kt

```diff
@@ -0,0 +1,64 @@
+package com.github.swent.echo.data
+
+import com.github.swent.echo.compose.map.MAP_CENTER
+import com.github.swent.echo.data.model.Association
+import com.github.swent.echo.data.model.BachelorSection
+import com.github.swent.echo.data.model.BachelorSemester
+import com.github.swent.echo.data.model.Event
+import com.github.swent.echo.data.model.Location
+import com.github.swent.echo.data.model.UserProfile
+import java.time.ZonedDateTime
+
+// This should come from the repository
+
+val SAMPLE_EVENTS: List<Event> =
```

Comment:
I guess these `SAMPLE_EVENTS` is just temporary until the repository is able to return events. Maybe create an issue so we don't forget to adapt and remove these samples.

Reply:
Added this to a new issue: #100.